oscilloscope/agilent.py

```diff
@@ -0,0 +1,195 @@
+"""
+Wrappers for Agilent oscilloscopes.
+
+"""
+
+
+from __future__ import absolute_import
+import oscilloscope.wrappers as wrappers
+import labrad.units as U
+import struct
+import numpy as np
+from twisted.internet.defer import inlineCallbacks, returnValue
+
+
+class AgilentDSOX4104AWrapper(wrappers.OscilloscopeWrapper):
```

Comment:
When working on the Infiniium Scope Server, I noticed that the scope supports more descriptive GPIB commands.

i.e. sending the scope ":WAV:STR ON" is equivalent to "WAVeform:STReaming ON" or ":TIMebase:REFerence CENTer" etc.  This is a nice form of self-documentation that we might want to start using if other scope's support it.  As another note, writing commands in this form also makes finding them in the scope's documentation easier as the programming guide always writes the commands in this longer format.